### DISCUSSION TITLE
An informal grammar engineering meeting next Monday online

### DISCUSSION THREAD

**Discussion by olzama** [delph-in/docs#88] [Team Posts]
Dear all,

I have long been wanting to start a regular grammar engineering meeting, to discuss various issues, from linguistic analysis to robustness, CALL applications and anything else that might be relevant for the participants. The goal would be to get feedback on what we are doing.

I don't know about a regular meeting at this moment, but @arademaker and I are going to meet via google meets next Monday November 14th 18:00 Madrid time (17:00 UTC) to discuss grammar matters, such as what is needed to make grammars more robust. Specifically, I will summarize what kind of tasks I have while reviving the SRG and updating it to work with ACE etc. Then we might discuss other matters. Anyone interested is welcome to join and bring matters to discuss!

meet.google.com/xui-uyxb-rsv

See you there, maybe! Also please feel free to leave any comments here regarding whether you would welcome such a meeting on a regular basis or what not.

**Comment by fcbond** [comment]
Hi,

I think it is a great idea, and will try to attend.

On Tue, 8 Nov 2022 at 18:10, Olga Zamaraeva ***@***.***>
wrote:

> Dear all,
>
> I have long been wanting to start a regular grammar engineering meeting,
> to discuss various issues, from linguistic analysis to robustness, CALL
> applications and anything else that might be relevant for the participants.
>
> I don't know about a regular meeting at this moment, but @arademaker
> <https://github.com/arademaker> and I are going to meet via google meets
> next Monday 18:00 Madrid time (17:00 UTC) to discuss grammar matters, such
> as what is needed to make grammars more robust. Specifically, I will
> summarize what kind of tasks I have while reviving the SRG and updating it
> to work with ACE etc. Then we might discuss other related matters. Anyone
> interested is welcome to join and bring matters to discuss!
>
> meet.google.com/xui-uyxb-rsv
>
> See you there, maybe! Also please feel free to leave any comments here
> regarding whether you would welcome such a meeting on a regular basis or
> what not.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/delph-in/teams/participants/discussions/36>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAIPZRQFMN5APS6CFLXMQSDWHKCRNANCNFSM6AAAAAAR2RSJ6I>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
Francis Bond <https://fcbond.github.io/>

**Comment by danflick** [comment]
I will also be glad to attend, but may come into the discussion a little late on Monday.

**Comment by emilymbender** [comment]
Thanks, Olga! This was fun. I look forward to future iterations :)

**Comment by arademaker** [comment]
Thank you all for collaborating with our initiative!

**Comment by olzama** [comment]
I think it would be great to have these meetings say every 2 weeks (maybe with a varying group of participants), and I'd be happy to commit to be the host however I do not have a professional zoom account (and we decided yesterday that Zoom was by far better than Google meets). I could host via Microsoft Teams (which is not so bad), or maybe someone else could host in Zoom? But Zoom does not allow you to assign co-hosts who are not from your institution, as far as I remember, and that could be a problem if I am the de facto host. Thoughts? Is there anyone out there who could commit to appearing biweekly, if only to start the meeting? :) Or would Teams be OK.

**Comment by wsun106** [comment]
Hi,


I have a professional zoom account and would like to host biweekly. 


Weiwei

**Comment by olzama** [comment]
Hi @wsun106 ! That would be amazing, thank you so much! Would Mondays at 17:00 London time (what we did yesterday) work for you, or would you like to propose another time slot?

**Comment by wsun106** [comment]
Hi @Olga 17.00 works for me; 17.30 is better ;)

**Comment by olzama** [comment]
Hi Weiwei @wsun106 , how about we schedule it every other Monday at 17:30 London time then? I can commit to being there more or less every time, so long as you start the meeting. So if some day you are busy and have to leave after starting it, I will still be there in case other people join.

**Comment by wsun106** [comment]
Great!

**Comment by arademaker** [comment]
I can also be there biweekly

**Comment by arademaker** [comment]
As you guys can see, GitHub is not good in the interpretation of replies from emails. I had to clean up the messages from @wsun106
